src/components/messages/ThreadSelector/Thread/Thread.scss

```diff
@@ -1,3 +1,85 @@
+// .Thread {
```

Comment:
Why do we have commented code ?

Please remove it if it's unused.

Reply:
Okay! Slipped my mind that I'd commented it there, ill get rid of it